### DISCUSSION TITLE
RP2040 host bare_api sample

### DISCUSSION THREAD

**Discussion by sonik-br** [adafruit/Adafruit_TinyUSB_Arduino#302] [Q&A]
I'm trying to use the [bare_api](https://github.com/hathach/tinyusb/blob/master/examples/host/bare_api/src/main.c) sample.
It shows the device descriptor and shows the message "Listen to [dev 1: ep 84]"
But it does not display any data report.

This sample is not compatible with this version of the lib?

**Comment by hathach** [comment]
that is example on core tinyusb, please make discussion there.

**Reply by sonik-br** [reply]
I understand.
I'm asking here as the pico [pio-usb](https://github.com/sekigon-gonnoc/Pico-PIO-USB/blob/main/examples/arduino/device_info/device_info.ino) lib for arduino uses this repo "Adafruit_TinyUSB_Arduino".
I wanted to use the bare_api sample but looks like it's not compatible with this version of tinyusb.
